src/app/_components/Specialists/DistrictFilter.js

```diff
@@ -52,19 +51,23 @@ function DistrictList() {
       </ul>
       <ClearFilterButton
         clear={() => {
-          remove();
+          districtParam.remove();
         }}
       />
     </>
   );
 }
 
+DistrictList.propTypes = {
+  districtsInUrl: PropTypes.arrayOf(PropTypes.string),
+};
+
 export function DistrictFilter() {
   const districtsInUrl = useSearchParams().getAll('district');
 
   return (
     <FilterBase filterText="Райони" count={districtsInUrl?.length || 0}>
-      <DistrictList />
+      <DistrictList districtsInUrl={districtsInUrl || []} />
```

Comment:
What do you think about assigning to specializationsInUrl empty array in case of `useSearchParams().getAll('district');` return nullable value?
in this case, you also do not have to check the length of the array on the line above

And also please do the same refactoring to other places in this PR

Reply:
thanks, will do